internal/fakecgo/doc.go

```diff
@@ -30,4 +30,3 @@
 package fakecgo
 
 //go:generate go run gen.go
-//go:generate gofmt -s -w symbols.go
```

Comment:
Just to confirm: Don't we need this?

Reply:
Correct it's formatted using `go/format` in `gen.go`